docs/CHANGELOG.md

```diff
@@ -29,7 +29,7 @@ New ENV vars:
 #### Bootstrap job
 
 Added a new `bootstrap` job type. This job removes the need for every job to implement their own bootstrapping logic.
-It makes OCR2 jobs that currently use `isBootstrapPeer=true` deprecated but still supported.
+It makes OCR2 jobs that currently use `isBootstrapPeer=true` deprecated and migrates those jobs to the new format.
```

Comment:
"deprecated" is usually interpreted as "it still works but might not work in the future"

I think we wanna just say they are migrated.